src/handbook/sales/demo.md

```diff
@@ -1,33 +1,118 @@
 ## Demo
 
-Sales demo's are vital for customer to gain a deep understanding in the value
-offered by the FlowForge product.
+This document explains the process of delivering a sales demo of the features of FlowForge. It is not intended to cover all features but to give potential customers a good understanding of FlowForge's value.
 
-### FlowForge Basics
+### Setup (complete before the demo)
 
-To develop
+Before we begin a sales demo we need to set up an environment.
 
-### Node-RED Basics
+[Create a coupon](https://flowforge.com/handbook/operations/billing/#coupons) with the value of $50usd, note down the coupon code. 
 
-To develop
+Make sure you are already logged into your flowforge account (@flowforge.com email) on production, not as an administrator.
 
-### Deployments
+Create a new team for this demo, you need to do this before the demo so that you don’t have to show your credit card details on screen. Use the coupon to assign your new team the $50 credit.
 
-#### Setup steps
+Create one instance called ‘central’, leave the rest to be created in the demo.
 
-1. Request access to the device_demo group on FlowForge
-1. Ensure you've got an up-to-date device agent: `npm install @flowforge/flowforge-device-agent`
+Make sure you have an example device ready, [install the latest device agent](https://flowforge.com/docs/user/devices/).
 
-#### Demo
 
-1. Go to the teams "Devices" page, register a new device. 
-1. Download the YAML file per the instructions
+Make sure the device agent is running before you start the demo.
 
-#### Explain
+The script should not be followed word for word, it's more a suggestion of what to say which fits with what will be displayed. Be natural, use your own words.
 
-1. FlowForge allows customers to manage many Node-RED instances, where ever these
-are running. The only requirement for them is to be (intermittently) connected
-to the internet and be able to run Node-RED.
-1. Devices assigned to a project will run the same flows
-1. Devices can be differentiated from one-another by using environment variables
+### Script
 
+FlowForge makes it easy for teams of developers to collaborate on creating, maintaining, and improving applications built in Node-RED.
+
+It can manage instances of Node-RED running on FlowForge Cloud, a version of FlowForge hosted by yourselves, and on your own hardware, for example devices running sensors on your shop floor.
+
+In this demo we're going to focus on the key features of FlowForge, there is a lot that we're not going to cover to keep this suitably brief but we will have some time after the demo for any questions.
+
+#### Key Concepts
```

Comment:
Given we have [the docs](https://flowforge.com/docs/user/concepts/), this should be a list IMO?